.riot/requirements/4211915.txt

```diff
@@ -1,54 +0,0 @@
-#
```

Comment:
If these files got deleted, have not others been created for the new dependencies?